service/src/test/java/dk/dbc/weekresolver/service/WeekResolverResourceIT.java

```diff
@@ -0,0 +1,160 @@
+package dk.dbc.weekresolver.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.Objects;
+
+import dk.dbc.httpclient.HttpGet;
+import dk.dbc.weekresolver.connector.WeekResolverConnector;
+import dk.dbc.weekresolver.connector.WeekResolverConnectorException;
+import dk.dbc.weekresolver.model.WeekCodeFulfilledResult;
+import dk.dbc.weekresolver.model.WeekResolverResult;
+import dk.dbc.weekresolver.model.YearPlanFormat;
+import dk.dbc.weekresolver.model.YearPlanResult;
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.Test;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class WeekResolverResourceIT extends AbstractWeekresolverServiceContainerTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(WeekResolverResourceIT.class);
+
+    @Test
+    void openapi() {
+        final HttpGet httpGet = new HttpGet(httpClient)
+                .withBaseUrl(weekresolverServiceBaseUrl)
+                .withPathElements("openapi");
+
+        final Response response = httpClient.execute(httpGet);
+        assertThat("status code", response.getStatus(), is(200));
+        final String openapi = response.readEntity(String.class);
+        assertThat("openapi", openapi, containsString("Provides weekcode calculations"));
+    }
```

Comment:
Hmm skal vi teste for den? Og tester den ikke bare om openapi docs er der?  Man kan vel ikke rigtig teste om de er rigtige?